docs/source/upgrading_your_components.md

```diff
@@ -0,0 +1,262 @@
+# Upgrading your components
+
+*Audience: network administrators, node administrators*
+
+For information about special considerations for the latest release of Fabric, check out [Upgrading to the latest release of Fabric](./upgrade_to_newest_version.html).
+
+This topic will only cover the process for upgrading components. For information about how to edit a channel to change the capability level of your channels, check out [Updating a channel capability](./updating_capabilities.html).
+
+Note: when we use the term “upgrade” in Hyperledger Fabric, we’re referring to changing the version of a component (for example, going from one version of a binary to the next version). The term “update,” on the other hand, refers not to versions but to configuration changes, such as updating a channel configuration or a deployment script. As there is no data migration, technically speaking, in Fabric, we will not use the term "migration" or "migrate" here.
+
+## Overview
+
+At a high level, upgrading the binary level of your nodes is a two step process:
+
+1. Backup the ledger and MSPs.
+2. Upgrade binaries to the latest version.
+
+If you own both ordering nodes and peers, it is a best practice to upgrade the ordering nodes first. If a peer falls behind or is temporarily unable to process certain transactions, it can always catch up. If enough ordering nodes go down, by comparison, a network can effectively cease to function.
+
+This topic presumes that these steps will be performed using Docker CLI commands. If you are utilizing a different deployment method (Rancher, Kubernetes, OpenShift, etc) consult their documentation on how to use their CLI.
+
+Note that you will also need to update the YAML configuration file for the nodes (for example, the `orderer.yaml` file) with the one from the release artifacts.
+
+To do this, backup the `orderer.yaml` or `core.yaml` file (for the peer) and replace it with the `orderer.yaml` or `core.yaml` file from the release artifacts. Then port any modified variables from the backed up `orderer.yaml` or `core.yaml` to the new one. Using a utility like `diff` may be helpful. Note that updating the YAML file from the release rather than updating your old YAML file **is the recommended way to update your node YAML files**, as it reduces the likelihood of making errors.
```

Comment:
Since our upgrade instructions assumes that you are using the docker images, the yaml files with latest defaults are baked into the images, and any overrides are typically done by using environment variables as the next section suggests. Therefore you could avoid all the discussion of yaml files throughout this topic, and have a single note somewhere that says if you are using native binaries instead of docker images, then you will need manage the yaml files as described here.